experimental/devices/mfrc522/example_test.go

```diff
@@ -23,7 +23,7 @@ func Example() {
 		log.Fatal(err)
 	}
 
-	// Using SPI as an example. See package ./spi/spireg for more details.
+	// Using SPI as an example. See package "periph.io/x/periph/conn/spi/spireg" for more details.
 	p, err := spireg.Open("")
 	if err != nil {
 		log.Fatal(err)
```

Comment:
Would it help if you added a LimitSpeed() call here?

Reply:
Yes that's what I did in my local test but I don't think I should add it to he example because I think the frequency issue is related to the card I'm using. I have a regular Mifare Classic and it work perfectly with the default 10 Mhz frequency. However, with my Desfire ev1, it doesn't work without lowering the frequency to 100 Khz.

Reply:
You could add something like this?

```
// If the device is misbehaving, you can try limiting the communication speed:
if err := p.LimitSpeed(1*physics.MegaHertz); err != nil {
  log.Fatal(err)
}
```

Reply:
Good idea, I will add that later.